### PR TITLE
ContractKit v0.4.18 Release

### PR DESCRIPTION
**Pull request by codyborn**
### Description

Bump version of ContractKit to 0.4.19 for 0.4.18 release. Ignore the ODIS related changes as [that PR](https://github.com/celo-org/celo-monorepo/pull/5621) is still being merged. Will not merge this until the parent is first merged.

### CODE REVIEW

File: packages/contractkit/CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ Other Changes
 
 ## Published
 
+### **[0.4.18]** -- 2020-10-30
+Features
+- ODIS Client Update - [#5621](https://github.com/celo-org/celo-monorepo/pull/5621)]
+
```
Comment (medhakothari) [inline packages/contractkit/CHANGELOG.md:29]: I think #5270, #5091, #5624, & #5436 also need to be added here

**Review by medhakothari — CHANGES_REQUESTED** [review]
Looks great, just left one small comment + waiting on getting the code changes merged!

File: packages/contractkit/CHANGELOG.md (hunk 1 above)
Comment (codyborn) [reply packages/contractkit/CHANGELOG.md:29]: Added, thanks! Also added a comment to the Notion doc to do this check:
https://www.notion.so/DevX-Package-Release-Guide-8da831a90dc8414fab93653f0228057b

**Review by medhakothari — APPROVED** [review]
LGTM!

**Pull request merged**